projects/Array-Rotation/README.md

```diff
@@ -0,0 +1,62 @@
+---
+title: List/Array rotation in Python
+layout: default
+date: 2020-10-14
+last-modified: 2020-10-14
+featured-image: {FEATURED IMAGE NAME}
+tags: Array/ List
+authors:
+  - {AUTHOR USERNAME FROM _data/authors.yml}
```

Comment:
Make sure you add yourself to this file.

Reply:
hey, whereto find that authors.yml ? I didn't find one in the repo.

Reply:
The path is in that line: `_data/authors.yml`